index.js

```diff
@@ -48,6 +51,91 @@ class ExecutorQueue extends Executor {
         this.fuseBox = new FuseBox();
         this.fuseBox.addFuse(this.queueBreaker);
         this.fuseBox.addFuse(this.redisBreaker);
+
+        this.scheduler = new Resque.Scheduler({ connection: redisConnection });
+    }
+
+    /**
+     * Starts a new periodic build in an executor
+     * @method _startPeriodic
+     * @param {Object} config               Configuration
+     * @param {Object} config.pipeline      Pipeline of the job
+     * @param {Object} config.job           Job object to create periodic builds for
+     * @param {Object} config.tokenGen      Function to generate JWT from username, scope and scmContext
+     * @param {Boolean}config.update        Boolean to determine if updating existing periodic build
+     * @return {Promise}
+     */
+    async _startPeriodic(config) {
```

Comment:
`async _startPeriodic({pipeline, job, tokenGen, update = false})`
and you can remove the next 4 lines for initialization.